src/toobj.d

```diff
@@ -830,7 +830,7 @@ void toObjFile(Dsymbol ds, bool multiobj)
                 if (global.params.symdebug)
                     toDebug(sd);
 
-                genTypeInfo(sd.type, null);
+                //genTypeInfo(sd.type, null);
```

Comment:
It's probably better to remove this line than comment it out. It's still there in version control so we'll always be able to restore it if necessary.